cfme/cloud/provider.py

```diff
@@ -48,6 +48,7 @@
         ('ipaddress_text', Input("ipaddress"), {"removed_since": "5.4.0.0.15"}),
         ('region_select', {version.LOWEST: Select("select#provider_region"),
             "5.5": AngularSelect("provider_region")}),
+        ('amazon_region_select', AngularSelect("provider_region")),
```

Comment:
Ahh I see what's happened.....personally I like to try this
* Remove both ```amazon_region_select``` and ```azure_region_select``` and replace them with just a generic ```region_select``` this will mean fixing up the provider forms, but there is no point having the same field defined twice.